src/styles/main.scss

```diff
@@ -1,5 +1,6 @@
 @import "~bootstrap/scss/bootstrap";
 @import "~@fortawesome/fontawesome-free/css/all.min.css";
+@import "../javascripts/components/planes/planes.scss";
```

Comment:
let's import this in the planesList component instead of reaching down into there from here to grab it